app/imports/ui/pages/ListEvents.jsx

```diff
@@ -1,11 +1,52 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
-import HelpButton from '../components/HelpButton';
```

Comment:
Same as before, I'd like to keep HelpButton included. You can just put it as the last component before closing the container with </Container>